app/javascript/article-form/elements/moreConfig.jsx

```diff
@@ -1,8 +1,17 @@
 import { h, Component } from 'preact';
 import PropTypes from 'prop-types';
-import { generateMainImage } from '../actions';
 
-// const ImageManagement = ({ onExit }) => (
+const TextField = ({ label, id, value, onKeyUp }) => {
+  return (
+    <div>
+      <label htmlFor={id}>
+        {label}
+        <input type="text" value={value} name={id} onKeyUp={onKeyUp} id={id} />
+      </label>
+    </div>
+  );
+};
+
 export default class MoreConfig extends Component {
   constructor(props) {
     super(props);
```

Comment:
The constructor right now doesn't do anything, it can be removed

Reply:
One line commented in that constructor section, that's why i didn't removed previously. Now i removed it @rhymes